modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Event.scala

```diff
@@ -24,6 +24,8 @@ sealed trait SystemEvent
 case class Completed[R](id: Sequence.Id, i: Int, r: R) extends SystemEvent
 case class Failed[E](id: Sequence.Id, i: Int, e: E) extends SystemEvent
 case class Executed(id: Sequence.Id) extends SystemEvent
+case class Executing(id: Sequence.Id) extends SystemEvent
```

Comment:
Now they are going just as log events but we can send the state or something else if that useful for the client.